parachain/runtime/tests/mock/mod.rs

```diff
@@ -219,17 +226,38 @@ impl UserData {
     }
 }
 
-#[derive(Debug, PartialEq, Default, Clone)]
+#[derive(Debug, Default, Clone)]
 pub struct FeePool {
-    pub balance: u128,
-    pub tokens: u128,
+    pub vault_backing_rewards: u128,
+    pub vault_issuing_rewards: u128,
+    pub relayer_backing_rewards: u128,
+    pub relayer_issuing_rewards: u128,
+}
+
+fn abs_difference<T: std::ops::Sub<Output = T> + Ord>(x: T, y: T) -> T {
+    if x < y {
+        y - x
+    } else {
+        x - y
+    }
+}
+
+impl PartialEq for FeePool {
```

Comment:
Not too sure if I like this change.. On the one hand it is very convenient, but on the other hand we lose some accuracy in our tests. E.g. from the tests we might mistakenly conclude that no funds are ever lost